projects/mercury/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "scripts": {
     "dev": "yarn build && concurrently \"yarn localdevelopment\" \"yarn saturn\" \"yarn pluto\" \"yarn start\"",
+    "backend": "concurrently \"yarn localdevelopment\" \"yarn pluto\"",
```

Comment:
I wouldn't add these commands. This one is missing saturn, which is backend. Also it introduces another type of grouping of already existing groups, so makes it a bit over-complex.